api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/inntekt/InntektProducer.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.api.inntekt
+
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helsearbeidsgiver.felles.BehovType
+import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.json.toJson
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.publish
+import no.nav.helsearbeidsgiver.felles.serializers.LocalDateSerializer
+import no.nav.helsearbeidsgiver.felles.serializers.UuidSerializer
+import no.nav.helsearbeidsgiver.inntektsmelding.api.logger
+import java.util.UUID
+
+class InntektProducer(
+    private val rapid: RapidsConnection
+) {
+    init {
+        logger.info("Starter InntektProducer...")
+    }
+
+    fun publish(request: InntektRequest): UUID {
```

Comment:
This functionality is a good oportunity to be quit of Akkumulatoren. Vi kan prate på mandag at vi kan gjøre noe justeringer å bli kvitt av det.